2020-03-18 14:47:10,189 - C:/Users/15992/PythonProject/Test_Ui_Android/log/test_log/testlog.py[line:43] - ERROR: error
